src/modules/settings/components/settings/twitch/ScrollVideoSeek.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Panel from 'rsuite/Panel';
+import Toggle from 'rsuite/Toggle';
+import {registerComponent} from '../../Store.jsx';
+import {CategoryTypes, SettingIds} from '../../../../../constants.js';
+import styles from '../../../styles/header.module.css';
+import useStorageState from '../../../../../common/hooks/StorageState.jsx';
+
+function ScrollVideoSeek() {
+  const [value, setValue] = useStorageState(SettingIds.SCROLL_VIDEO_SEEK);
+
+  return (
+    <Panel header="Scroll Video Seek">
```

Comment:
Not entirely sure how you would describe this feature. `Seek by scrolling` maybe? Currently not happy with the name.

Reply:
Updated it